docs/storage-usage-quota.md

```diff
@@ -0,0 +1,79 @@
+---
+hide:
+- navigation
+
+license: |
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+---
+
+Storage Usage Quota Guide
+===
+This documentation contains how RSS limit user's storage resource usage by setting quota.
+
+## Quota limitation
+
+RSS has a configurable user storage quota system. This allows RSS admin to manage each user's
+max resource usage to keep RSS cluster more stable. This feature can avoid RSS cluster resources
+been occupied by several users with huge application.
```

Comment:
been -> being
several users -> the minority
application -> applications

Reply:
Done